library/index.c

```diff
@@ -323,24 +323,25 @@ int eblob_index_blocks_fill(struct eblob_base_ctl *bctl)
 }
 
 
-static struct eblob_disk_control *eblob_find_on_disk(struct eblob_backend *b,
-		struct eblob_base_ctl *bctl, struct eblob_disk_control *dc,
+static int eblob_find_on_disk(struct eblob_backend *b,
```

Comment:
It is possible to check whether interval contains key before index block reading, but additional pread may be needed anyway.